packages/odie-client/src/components/message/user-message.tsx

```diff
@@ -22,7 +22,9 @@ export const UserMessage = ( {
 
 	const hasCannedResponse = message.context?.flags?.canned_response;
 	const isRequestingHumanSupport = message.context?.flags?.forward_to_human_support;
-	const isOnlyMessage = message.context?.flags?.only_message;
+	const isOnlyMessage =
```

Comment:
This seems to extend the meaning of `isOnlyMessage` to something else like `isMessageWithoutEscalationOption`. Could we choose a variable name that is decriptive of what it actually is?